shared-scripts.sh

```diff
@@ -275,3 +275,184 @@ function _scanArtifacts {
         exit 1
     fi
 }
+
+function _putCredsInProfile {
+    mkdir -p ~/.aws
+    touch ~/.aws/config ~/.aws/credentials 
+    python3 codebuild_specs/sh-files/aws-configure-credentials.py
+}
+
+function _installIntegTestsDependencies {
+    apt-get update
+    apt-get install -y sudo
+    sudo apt-get install -y tcl
+    sudo apt-get install -y expect
+    sudo apt-get install -y zip
+    sudo apt-get install -y lsof
```

Comment:
low priority/nit, but we just added lsof/zip/expect to the base image so these shouldn't be required anymore; I think this might be leftover from before we had our own custom linux images